src/app/component/comments/components/comment-body/comment-body.component.ts

```diff
@@ -1,14 +1,31 @@
 import { Component, OnInit, Input } from '@angular/core';
 import { CommentsService } from '../../services/comments.service';
+import { UserOwnAuthService } from '@global-service/auth/user-own-auth.service';
+import set = Reflect.set;
```

Comment:
Do you really use it ? If not please remove it.

Reply:
> Do you really use it ? If not please remove it.

Fixed